client/src/components/Business/Business.jsx

```diff
@@ -606,6 +621,10 @@ function Business() {
                 soon to complete the verification process.
               </div>
             )}
+            {loadingImg ? <div className="loadingSign" style = {{ position: "fixed", top: "50%", left: "50%", transform: "translate(-50%, -50%)" }}>
+              <LoadingSign />
+            </div> : null 
+              }
```

Comment:
just a heads up that you can write this like this without `?` and `:` !
```suggestion
            {loadingImg  && (<div className="loadingSign" style = {{ position: "fixed", top: "50%", left: "50%", transform: "translate(-50%, -50%)" }}>
              <LoadingSign />
            </div>)
              }
```

Reply:
Thx for the great tip !!